### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by traylenator**
Release 5.0.0  

First version in voxpupuli space.

### CODE REVIEW

**Comment by traylenator** [comment]
Is there a way to not junk the old changelog?

**Comment by rwaffen** [comment]
please wait for #13

**Comment by rwaffen** [comment]
> Is there a way to not junk the old changelog?

move it to HISTORY.md?

**Comment by rwaffen** [comment]
13 is done, please rebase :)

File: HISTORY.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+
```
Comment (rwaffen) [inline HISTORY.md]: i think this as to be removed, only the releases should be in here. then gcg should acknowledge this file.

Comment (rwaffen) [reply HISTORY.md]: we have done this for example in https://github.com/voxpupuli/puppet-wildfly

**Review by rwaffen — APPROVED** [review]
hmm somehow the history does not work yet, but dont want to block the release

**Pull request merged**